sirepo/package_data/static/js/raydata.js

```diff
@@ -320,7 +320,10 @@ SIREPO.app.directive('scansTable', function() {
                     </tr>
                   </tbody>
                 </table>
-                <div ng-if="awaitingScans">Loading scans...</div>
+                <div style="height: 20px;">
+                  <div ng-if="awaitingScans" data-dots-animation="" data-text="Checking for new scans"></div>
+                  <div ng-if="noScansReturned">No matching scans returned.</div>
```

Comment:
What about `No scans found`? It is simpler. In addition matching implies the user can change their query which they can't really. And when I see "returned" it makes me think that maybe the data just didn't come back which isn't what's going on here.

Reply:
Also,  is this available on the "Executed" tab? I couldn't get it to appear.

Reply:
That's weird, it's appearing for me on the executed tab.

And sure, my initial thinking was that I wanted to make it clear that there were no scans matching the query, rather than there being no scans in the catalog. I think you're right that `No scans found` is clear enough though.

What do you mean by this though?

> In addition matching implies the user can change their query which they can't really.

There are no scans matching the date range and catalog, but they could change those which changes their query. Or am I misunderstanding?

Reply:
[07aefb1](https://github.com/radiasoft/sirepo/pull/5141/commits/07aefb105646ca785918a47cad0460fddb2d4854)

Reply:
> That's weird, it's appearing for me on the executed tab.

Ah my bad. I hadn't selected a date range so it wasn't querying. I see it now.

> There are no scans matching the date range and catalog, but they could change those which changes their query. Or am I misunderstanding?

Since I was only viewing on the Queued scans tab I wasn't thinking about the date range. The only thing the user could change was the catalog name which isn't much. This is 100% my own personal feeling so if you don't agree we can go with your text.